transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/TransactionOutboxImpl.java

```diff
@@ -111,23 +114,34 @@ public ParameterizedScheduleBuilder with() {
   @SuppressWarnings("UnusedReturnValue")
   @Override
   public boolean flush() {
+    return flush(false);
```

Comment:
Why not just do both flush types in the one call to `flush()`?

We just want to clear out all work. Why complicate it?

Reply:
The idea was to use separate `TransactionOutbox` instances for flushing ordered and non-ordered tasks because the scheduler params like a delay between iterations etc. (and flush frequency itself) are planned to be much fewer in the 1st case than in the 2nd. What do you think about this?

Reply:
Interesting. I'll ponder.